pytest.ini

```diff
@@ -1,10 +1,9 @@
 [pytest]
 addopts =
     --verbose --ignore=regression
-    --cov=autosubmit --cov-config=.coveragerc
-    --cov-report=html:test/htmlcov --cov-report=xml:test/coverage.xml
```

Comment:
Disabled coverage by default as that slows down the unit tests, and the debugger is not able to locate the lines with break points (as the code is modified by `coverage`).